reframe/frontend/statistics.py

```diff
@@ -229,6 +231,20 @@ def json(self, force=False):
         return self._run_data
 
     def print_failure_report(self, printer, rerun_info=True):
+        def first_n_lines_of_file(filename, prefix, num_lines=10):
```

Comment:
Let's be less verbose :-)

```suggestion
        def _head_n(filename, prefix, num_lines=10):
```